pset-functions.js

```diff
@@ -30,6 +30,35 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 1:')
 
 // Add your code below this line
+// logic for blackJack
+function blackJack(playerCardScore, dealerCardScore) {
+  // making sure we're playing poker
+  if (isNaN(parseInt(playerCardScore))|| isNaN(parseInt(dealerCardScore))) {
+    return "Doesn't look like you're playing Black Jack"
+  } else if (playerCardScore > 21 && dealerCardScore > 21) {
+    // if both players bust
+    return 0
+  } else if (playerCardScore > 21 && dealerCardScore <= 21) {
+    // if player busts
+    return dealerCardScore
+  } else if (playerCardScore <= 21 && dealerCardScore > 21) {
+    // if dealer busts
+    return playerCardScore
+  } else if (playerCardScore > dealerCardScore) {
```

Comment:
Nate, really great use of if/else statements here; I love that you considered an edge case where a user may not input a number as your first filter. The way you set up your if/else cascades perfectly for all potential scenarios in the proper order. Once you get here to line 47, you can also take advantage of built in Math methods that could help you avoid running additional if/else blocks (in this specific example). For a sample alternate solution using Math methods please check [here](https://github.com/jsd20200127/hw-02-functions-solution/blob/e87c29073cf7285a5045ec8708561f279474f2d2/pset-functions.js#L40-L41)